.riot/requirements/releasenotes/notes/3.12_support-422a62d1871877c2.yaml

```diff
@@ -0,0 +1,14 @@
+---
+prelude: >
+    We added support for Python 3.12 but (add caveats)
+
+features:
+  - |
+    Adds support for Python 3.12
```

Comment:
This should probably include a mention of dropped support for <3.8

Reply:
We added a separate release note for dropping support Python < 3.7 here: https://github.com/DataDog/dd-trace-py/blob/yunkim/add-py-312/releasenotes/notes/release-2.0-3af0045e2261bd02.yaml